### PR TITLE
Remove subset inputs

### PR DESCRIPTION
**Pull request by paigem**
Create 30-day time chunks (13 total, 12 x 30-day chunks and 1 x 5-day chunk) in attempt to solve #5 

@cisaacstern could you merge this PR? Thanks for your help with this!

### CODE REVIEW

**Comment by cisaacstern** [comment]
@paigem, since we ended up with an error in #5 which may have been caught by an initial test, I think it may save us some effort if I run a test prior to merging. I'll do that now, and merge pending a successful result. Thanks for the PR!

**Comment by cisaacstern** [comment]
Misspelled the last command!

**Comment by cisaacstern** [comment]
Hmm the linked logs are not helpful without a solution for https://github.com/pangeo-forge/pangeo-forge.org/issues/63

Pulling logs from the backend directly, I'm seeing this rather opaque error 

<details>

```
Task 'store_chunk[336]': Exception encountered during task execution!
Traceback (most recent call last):
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/comm/tcp.py", line 205, in read
    frames_nbytes = await stream.read_bytes(fmt_size)
tornado.iostream.StreamClosedError: Stream is closed

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/prefect/engine/task_runner.py", line 861, in get_task_run_state
    value = prefect.utilities.executors.run_task_with_timeout(
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/prefect/utilities/executors.py", line 323, in run_task_with_timeout
    return task.run(*args, **kwargs)  # type: ignore
  File "/usr/local/lib/python3.9/site-packages/registrar/flow.py", line 113, in wrapper
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/pangeo_forge_recipes/recipes/xarray_zarr.py", line 631, in store_chunk
    with lock_for_conflicts(lock_keys, timeout=config.lock_timeout):
  File "/srv/conda/envs/notebook/lib/python3.9/contextlib.py", line 119, in __enter__
    return next(self.gen)
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/pangeo_forge_recipes/utils.py", line 106, in lock_for_conflicts
    acquired = lock.acquire(timeout=timeout)
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/lock.py", line 137, in acquire
    result = self.client.sync(
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/client.py", line 868, in sync
    return sync(
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/utils.py", line 332, in sync
    raise exc.with_traceback(tb)
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/utils.py", line 315, in f
    result[0] = yield future
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/tornado/gen.py", line 762, in run
    value = future.result()
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/core.py", line 895, in send_recv_from_rpc
    result = await send_recv(comm=comm, op=key, **kwargs)
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/core.py", line 672, in send_recv
    response = await comm.read(deserializers=deserializers)
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/comm/tcp.py", line 221, in read
    convert_stream_closed_error(self, e)
  File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/comm/tcp.py", line 128, in convert_stream_closed_error
    raise CommClosedError(f"in {obj}: {exc}") from exc
distributed.comm.core.CommClosedError: in <TCP (closed) ConnectionPool.lock_acquire local=tcp://10.60.0.5:58998 remote=tcp://dask-jovyan-d630fd13-4.pangeo-forge-columbia-staging-bakery:8786>: Stream is closed
```

</details>

@paigem, does the version of the recipe in this PR execute locally for you without error?

**Comment by paigem** [comment]
Yes, @cisaacstern this recipe runs without errors locally.

Side note: the warning about 255 character length issues gets printed out many, many times in the local run, and so it's a bit difficult to sift through and see if there are any *other* warnings that show up. But, it looks like in this case, in addition to no errors, there are no other warnings either.

File: feedstock/recipe.py [hunk 1]
```diff
@@ -40,9 +40,9 @@ def make_filename(variable, time):
...
 
-target_chunks = {"time": 73} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
+target_chunks = {"time": 30} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
 
 recipe = XarrayZarrRecipe(
             file_pattern=pattern,
             target_chunks=target_chunks,
-            subset_inputs = {"time": 5}) # set 5 chunks per year, each with time of length 73 (to total 365)
\ No newline at end of file
+            subset_inputs = {"time": 13}) # with time chunks of 30, we need a total of 13 chunks (12x30-day chunks, 1x5-day chunk)
```
Comment (cisaacstern) [inline feedstock/recipe.py]:
```suggestion

```

Based on a closer reading of https://github.com/pangeo-forge/cesm-atm-025deg-feedstock/issues/5#issuecomment-1151287882, it's not actually clear to me that input subsetting is required for this recipe. Each NetCDF input is slightly less than 1 GB (according to the linked comment). Removing subset inputs will not necessarily fix the error we are seeing, but reducing then number of factors in play will help me focus on which aspect of the recipe may be causing the problem.

Comment (cisaacstern) [reply feedstock/recipe.py]: @paigem, I'm going to accept this change and then run another test, to see if we get a different result.

File: feedstock/recipe.py [hunk 2]
```diff
@@ -40,9 +40,8 @@ def make_filename(variable, time):
...
 
-target_chunks = {"time": 73} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
+target_chunks = {"time": 30} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
 
 recipe = XarrayZarrRecipe(
             file_pattern=pattern,
-            target_chunks=target_chunks,
-            subset_inputs = {"time": 5}) # set 5 chunks per year, each with time of length 73 (to total 365)
\ No newline at end of file
+            target_chunks=target_chunks,
```
Comment (cisaacstern) [inline feedstock/recipe.py]:
```suggestion
            target_chunks=target_chunks,
            )
```

File: feedstock/recipe.py [hunk 3]
```diff
@@ -40,9 +40,9 @@ def make_filename(variable, time):
 
 # Define the recipe
 
-target_chunks = {"time": 73} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
+target_chunks = {"time": 30} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
 
 recipe = XarrayZarrRecipe(
             file_pattern=pattern,
             target_chunks=target_chunks,
-            subset_inputs = {"time": 5}) # set 5 chunks per year, each with time of length 73 (to total 365)
```
Comment (cisaacstern) [inline feedstock/recipe.py:48]:
```suggestion
subset_inputs = {"time":  2},
```

Comment (cisaacstern) [inline feedstock/recipe.py:48]:
```suggestion
subset_inputs = {"time": 2},
```

Comment (cisaacstern) [inline feedstock/recipe.py:48]:
```suggestion
            subset_inputs = {"time": 2},
```

File: feedstock/recipe.py [hunk 4]
```diff
@@ -40,9 +40,9 @@ def make_filename(variable, time):
...
\ No newline at end of file
+            )
```
Comment (cisaacstern) [inline feedstock/recipe.py:48]:
```suggestion
            subset_inputs = {"time": 2},
            )
```

**Comment by cisaacstern** [comment]
Two reflections:

1. I'd previously removed `subset_inputs` but having just taken a closer look at the input file sizes (which are each ~1 GB), I do think a `subset_inputs` make sense, and I've restored it to `{"time": 2}`, which is a more modest level of subsetting than we currently have in `main`.
2. I've just manually deleted the cached files for this recipe. It's occurred to me that my assumptions about cache-reuse with our current infrastructure may have been faulty: it's possible that subsequent recipe runs which attempt to re-use a cache from a prior recipe run may run into problems. We have a clean slate now for the cache.

**Comment by cisaacstern** [comment]
I've just increased worker memory in hopes that this may allow us to move past the KilledWorker issues we've seen. I'm going to try to re-run this recipe from the existing recipe run now.

**Comment by cisaacstern** [comment]
This hasn't officially failed yet, but it does appear to be stalled and likely to time out.

Because we have a lot more memory available to us now, I'm going to remove the subset inputs again to make this more intuitive to debug.

File: feedstock/recipe.py [hunk 5]
```diff
@@ -40,9 +40,10 @@ def make_filename(variable, time):
...
 
-target_chunks = {"time": 73} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
+target_chunks = {"time": 30} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
 
 recipe = XarrayZarrRecipe(
             file_pattern=pattern,
             target_chunks=target_chunks,
-            subset_inputs = {"time": 5}) # set 5 chunks per year, each with time of length 73 (to total 365)
\ No newline at end of file
+            subset_inputs = {"time": 2},
```
Comment (cisaacstern) [inline feedstock/recipe.py]:
```suggestion
```

**Comment by cisaacstern** [comment]
`399` has not failed yet, but I expect it will.

I'm going to run `501` now, which I imagine will also fail, but because we have no subsetting set (thanks to the larger memory allocation), it should be more intuitive to debug once it does.

**Comment by cisaacstern** [comment]
I just manually cancelled `501`. It had been hanging on a single `store_chunk` task since yesterday.

There's a lot of noise in the full traceback, but if you look closely, it appears to have been stalled on [acquiring a lock](https://github.com/pangeo-forge/pangeo-forge-recipes/blob/e764542d05261a0b31916c61cc3afe2a96608b81/pangeo_forge_recipes/utils.py#L106)

Note these lines present in the full trace

```
  |   | 2022-06-24T16:41:39.820282266Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/pangeo_forge_recipes/utils.py", line 106, in lock_for_conflicts
  |   | 2022-06-24T16:41:39.820286606Z stderr F     acquired = lock.acquire(timeout=timeout)
```

<details>

<summary>Traceback</summary>

```

2022-06-24T16:41:38.38910185Z stderr F distributed.dask_worker - INFO - Exiting on signal 15
--
  |   | 2022-06-24T16:41:38.389540753Z stderr F distributed.nanny - INFO - Closing Nanny at 'tcp://10.60.8.7:36169'
  |   | 2022-06-24T16:41:38.394733293Z stderr F distributed.worker - INFO - Stopping worker at tcp://10.60.8.7:44651
  |   | 2022-06-24T16:41:39.819390004Z stdout F [2022-06-24 16:41:39+0000] ERROR - prefect.CloudTaskRunner \| Task 'store_chunk[11]': Exception encountered during task execution!
  |   | 2022-06-24T16:41:39.819439877Z stdout F Traceback (most recent call last):
  |   | 2022-06-24T16:41:39.81944926Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/comm/tcp.py", line 205, in read
  |   | 2022-06-24T16:41:39.819462077Z stdout F     frames_nbytes = await stream.read_bytes(fmt_size)
  |   | 2022-06-24T16:41:39.819468248Z stdout F tornado.iostream.StreamClosedError: Stream is closed
  |   | 2022-06-24T16:41:39.819478403Z stdout F
  |   | 2022-06-24T16:41:39.819483755Z stdout F The above exception was the direct cause of the following exception:
  |   | 2022-06-24T16:41:39.819489907Z stdout F
  |   | 2022-06-24T16:41:39.81949485Z stdout F Traceback (most recent call last):
  |   | 2022-06-24T16:41:39.819500297Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/prefect/engine/task_runner.py", line 861, in get_task_run_state
  |   | 2022-06-24T16:41:39.819506001Z stdout F     value = prefect.utilities.executors.run_task_with_timeout(
  |   | 2022-06-24T16:41:39.819512129Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/prefect/utilities/executors.py", line 323, in run_task_with_timeout
  |   | 2022-06-24T16:41:39.819517295Z stdout F     return task.run(*args, **kwargs)  # type: ignore
  |   | 2022-06-24T16:41:39.819522842Z stdout F   File "/usr/local/lib/python3.9/site-packages/registrar/flow.py", line 113, in wrapper
  |   | 2022-06-24T16:41:39.819528814Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/pangeo_forge_recipes/recipes/xarray_zarr.py", line 631, in store_chunk
  |   | 2022-06-24T16:41:39.819533792Z stdout F     with lock_for_conflicts(lock_keys, timeout=config.lock_timeout):
  |   | 2022-06-24T16:41:39.819539175Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/contextlib.py", line 119, in __enter__
  |   | 2022-06-24T16:41:39.819544444Z stdout F     return next(self.gen)
  |   | 2022-06-24T16:41:39.819549625Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/pangeo_forge_recipes/utils.py", line 106, in lock_for_conflicts
  |   | 2022-06-24T16:41:39.819554837Z stdout F     acquired = lock.acquire(timeout=timeout)
  |   | 2022-06-24T16:41:39.819559897Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/lock.py", line 137, in acquire
  |   | 2022-06-24T16:41:39.819565039Z stdout F     result = self.client.sync(
  |   | 2022-06-24T16:41:39.819569999Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/client.py", line 868, in sync
  |   | 2022-06-24T16:41:39.81957527Z stdout F     return sync(
  |   | 2022-06-24T16:41:39.819582053Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/utils.py", line 332, in sync
  |   | 2022-06-24T16:41:39.819587125Z stdout F     raise exc.with_traceback(tb)
  |   | 2022-06-24T16:41:39.819591942Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/utils.py", line 315, in f
  |   | 2022-06-24T16:41:39.819596916Z stdout F     result[0] = yield future
  |   | 2022-06-24T16:41:39.819619038Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/tornado/gen.py", line 762, in run
  |   | 2022-06-24T16:41:39.819623473Z stdout F     value = future.result()
  |   | 2022-06-24T16:41:39.819627811Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/core.py", line 895, in send_recv_from_rpc
  |   | 2022-06-24T16:41:39.819633516Z stdout F     result = await send_recv(comm=comm, op=key, **kwargs)
  |   | 2022-06-24T16:41:39.819638537Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/core.py", line 672, in send_recv
  |   | 2022-06-24T16:41:39.819642915Z stdout F     response = await comm.read(deserializers=deserializers)
  |   | 2022-06-24T16:41:39.819659697Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/comm/tcp.py", line 221, in read
  |   | 2022-06-24T16:41:39.819664017Z stdout F     convert_stream_closed_error(self, e)
  |   | 2022-06-24T16:41:39.819668568Z stdout F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/comm/tcp.py", line 128, in convert_stream_closed_error
  |   | 2022-06-24T16:41:39.819672914Z stdout F     raise CommClosedError(f"in {obj}: {exc}") from exc
  |   | 2022-06-24T16:41:39.819677507Z stdout F distributed.comm.core.CommClosedError: in <TCP (closed) ConnectionPool.lock_acquire local=tcp://10.60.8.7:57734 remote=tcp://dask-jovyan-779c8f5b-c.pangeo-forge-columbia-staging-bakery:8786>: Stream is closed
  |   | 2022-06-24T16:41:39.820138647Z stderr F ERROR:prefect.CloudTaskRunner:Task 'store_chunk[11]': Exception encountered during task execution!
  |   | 2022-06-24T16:41:39.820186395Z stderr F Traceback (most recent call last):
  |   | 2022-06-24T16:41:39.820196449Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/comm/tcp.py", line 205, in read
  |   | 2022-06-24T16:41:39.820203736Z stderr F     frames_nbytes = await stream.read_bytes(fmt_size)
  |   | 2022-06-24T16:41:39.820211269Z stderr F tornado.iostream.StreamClosedError: Stream is closed
  |   | 2022-06-24T16:41:39.820217195Z stderr F
  |   | 2022-06-24T16:41:39.820223646Z stderr F The above exception was the direct cause of the following exception:
  |   | 2022-06-24T16:41:39.820228905Z stderr F
  |   | 2022-06-24T16:41:39.82023335Z stderr F Traceback (most recent call last):
  |   | 2022-06-24T16:41:39.820238096Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/prefect/engine/task_runner.py", line 861, in get_task_run_state
  |   | 2022-06-24T16:41:39.820244554Z stderr F     value = prefect.utilities.executors.run_task_with_timeout(
  |   | 2022-06-24T16:41:39.820249631Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/prefect/utilities/executors.py", line 323, in run_task_with_timeout
  |   | 2022-06-24T16:41:39.820254158Z stderr F     return task.run(*args, **kwargs)  # type: ignore
  |   | 2022-06-24T16:41:39.820258797Z stderr F   File "/usr/local/lib/python3.9/site-packages/registrar/flow.py", line 113, in wrapper
  |   | 2022-06-24T16:41:39.820263903Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/pangeo_forge_recipes/recipes/xarray_zarr.py", line 631, in store_chunk
  |   | 2022-06-24T16:41:39.820268316Z stderr F     with lock_for_conflicts(lock_keys, timeout=config.lock_timeout):
  |   | 2022-06-24T16:41:39.820272887Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/contextlib.py", line 119, in __enter__
  |   | 2022-06-24T16:41:39.82027737Z stderr F     return next(self.gen)
  |   | 2022-06-24T16:41:39.820282266Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/pangeo_forge_recipes/utils.py", line 106, in lock_for_conflicts
  |   | 2022-06-24T16:41:39.820286606Z stderr F     acquired = lock.acquire(timeout=timeout)
  |   | 2022-06-24T16:41:39.820291086Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/lock.py", line 137, in acquire
  |   | 2022-06-24T16:41:39.820295475Z stderr F     result = self.client.sync(
  |   | 2022-06-24T16:41:39.820299846Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/client.py", line 868, in sync
  |   | 2022-06-24T16:41:39.820304218Z stderr F     return sync(
  |   | 2022-06-24T16:41:39.820320311Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/utils.py", line 332, in sync
  |   | 2022-06-24T16:41:39.820324877Z stderr F     raise exc.with_traceback(tb)
  |   | 2022-06-24T16:41:39.820329103Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/utils.py", line 315, in f
  |   | 2022-06-24T16:41:39.820333453Z stderr F     result[0] = yield future
  |   | 2022-06-24T16:41:39.820337847Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/tornado/gen.py", line 762, in run
  |   | 2022-06-24T16:41:39.820342096Z stderr F     value = future.result()
  |   | 2022-06-24T16:41:39.820346316Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/core.py", line 895, in send_recv_from_rpc
  |   | 2022-06-24T16:41:39.820351913Z stderr F     result = await send_recv(comm=comm, op=key, **kwargs)
  |   | 2022-06-24T16:41:39.820356729Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/core.py", line 672, in send_recv
  |   | 2022-06-24T16:41:39.820361286Z stderr F     response = await comm.read(deserializers=deserializers)
  |   | 2022-06-24T16:41:39.82036562Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/comm/tcp.py", line 221, in read
  |   | 2022-06-24T16:41:39.820370048Z stderr F     convert_stream_closed_error(self, e)
  |   | 2022-06-24T16:41:39.820374447Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/site-packages/distributed/comm/tcp.py", line 128, in convert_stream_closed_error
  |   | 2022-06-24T16:41:39.820378986Z stderr F     raise CommClosedError(f"in {obj}: {exc}") from exc
  |   | 2022-06-24T16:41:39.820383506Z stderr F distributed.comm.core.CommClosedError: in <TCP (closed) ConnectionPool.lock_acquire local=tcp://10.60.8.7:57734 remote=tcp://dask-jovyan-779c8f5b-c.pangeo-forge-columbia-staging-bakery:8786>: Stream is closed
  |   | 2022-06-24T16:41:39.995283823Z stderr F distributed.nanny - WARNING - Worker process still alive after 1 seconds, killing
  |   | 2022-06-24T16:41:39.997992272Z stderr F distributed.dask_worker - INFO - End worker
  |   | 2022-06-24T16:41:39.999204802Z stderr F distributed.process - INFO - reaping stray process <SpawnProcess name='Dask Worker process (from Nanny)' pid=313 parent=1 started daemon>
  |   | 2022-06-24T16:41:40.196608195Z stderr F Exception in thread AsyncProcess Dask Worker process (from Nanny) watch process join:
  |   | 2022-06-24T16:41:40.19677442Z stderr F Traceback (most recent call last):
  |   | 2022-06-24T16:41:40.196795839Z stderr F   File "/srv/conda/envs/notebook/lib/python3.9/threading.py", line 973, in _bootstrap_inner
```

</details>

I'm going to try restoring the target_chunks to original `{"time": 73}`, which as a divisor of 365 (the number of time steps per file), should prevent the need for any locking.

This is the setting we had initially run with, and which caused a KilledWorker in #5, but we have 3x the worker memory now, so perhaps all will work smoothly.

File: feedstock/recipe.py [hunk 6]
```diff
@@ -40,9 +40,9 @@ def make_filename(variable, time):
 
 # Define the recipe
 
-target_chunks = {"time": 73} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
+target_chunks = {"time": 30} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
```
Comment (cisaacstern) [inline feedstock/recipe.py]:
```suggestion
target_chunks = {"time": 73} # Full spatial domain: "lat": 768, "lon": 1152, --> if not specified, will include full extent
```

**Comment by cisaacstern** [comment]
Ok so here was what ultimately happened, to make this work:

1. Increase worker memory from 4 GB -> 12 GB
2. Remove subsetting inputs (not required with more worker memory)
3. Keep original large time chunks, to avoid locking stalls (also possible via more memory)

Going to rename this PR accordingly, for clarity, then merge, which will trigger the next production build.

**Comment by rabernat** [comment]
Charles thanks so much for your perseverance here! 🙏

**Comment by paigem** [comment]
Amazing!! We have a successful test! Thank you @cisaacstern!!

**Comment by cisaacstern** [comment]
Screen Shot 2022-06-24 at 2 28 19 PM

KilledWorker on the production run 😵‍💫 https://pangeo-forge.org/dashboard/recipe-run/503

I will look into this more closely and get back with some ideas. Thanks for your patience, Paige.

**Comment by paigem** [comment]
Oh no!! So sorry about this @cisaacstern. Thanks for continuing to push this through!

For now, having the test data might go pretty far in the short term, so that's at least a good step!

**Pull request merged**